planning/standards-status.md

```diff
@@ -0,0 +1,83 @@
+| [IRCv3 Echo Message](https://ircv3.net/specs/extensions/echo-message-3.2) | &#10004; |     N/A       |                            |         |
+| [IRCv3 Multi-Prefix](https://ircv3.net/specs/extensions/multi-prefix-3.1) | &#10004; |     N/A       |                            |         |
+| [IRCv3 Server Time](https://ircv3.net/specs/extensions/server-time-3.2)  |  &#10004; |     N/A       |                            |         |
+| [XMPP](https://datatracker.ietf.org/doc/rfc3920)                   | &#10004;       |      N/A       |                            | XMPP is actually over 4 RFCs: [RFC3920](https://datatracker.ietf.org/doc/rfc3920), [RFC3921](https://datatracker.ietf.org/doc/rfc3921), [RFC6120](https://datatracker.ietf.org/doc/rfc6120), and [RFC6121](https://datatracker.ietf.org/doc/rfc6121) |
+| [XEP-0030](https://xmpp.org/extensions/xep-0030.html)              |  &#10004;      |      N/A       |                            |         |
+| [XEP-0045](https://xmpp.org/extensions/xep-0045.html)              |  &#10004;      |      N/A       |                            |         |
+| [XEP-0054](https://xmpp.org/extensions/xep-0054.html)              |  &#10004;      |      N/A       |                            |         |
+| [XEP-0059](https://xmpp.org/extensions/xep-0059.html)              |  &#10004;      |      N/A       |                            |         |
+| [XEP-0078](https://xmpp.org/extensions/xep-0078.html)              |  &#10004;      |      N/A       |                            |         |
+| [XEP-0085](https://xmpp.org/extensions/xep-0085.html)              |  &#10004;      |      N/A       |                            | Typing notifications |
+| [XEP-0092](https://xmpp.org/extensions/xep-0092.html)              |  &#10004;      |      N/A       |                            |         |
+| [XEP-0199](https://xmpp.org/extensions/xep-0199.html)              |  &#10004;      |      N/A       |                            |         |
+| [XEP-0203](https://xmpp.org/extensions/xep-0203.html)              |  &#10004;      |      N/A       |                            |         |
+| [XEP-0245](https://xmpp.org/extensions/xep-0245.html)              |  &#10004;      |      N/A       |                            |         |
+| [XEP-0249](https://xmpp.org/extensions/xep-0249.html)              |  &#10004;      |      N/A       |                            |         |
+| [XEP-0280](https://xmpp.org/extensions/xep-0280.html)              |  &#10004;      |      N/A       |                            |         |
+| [XEP-xxxx MUC Avatars](https://xmpp.org/extensions/inbox/muc-avatars.html) | &#10004; |    N/A       |                            |         |
+| [Matrix](https://spec.matrix.org/latest/client-server-api/)        |  &#10004;      |      N/A       | Donated to Matrix Foundation | UI only partially supports the full body of the specificaiton, not tracking on an MSC basis at this time |
```

Comment:
What does "Donated to Matrix Foundation" mean?

I'm involved in the Matrix specification though. 🤷

Reply:
We're also an ecpsystem member now, which is probably the bit to call out?

Is your spec core team seat "in the name of tb"?

Reply:
I updated the line to point to our Ecosystem Membership.

Reply:
> We're also an ecosystem member now, which is probably the bit to call out?

Right, which is actually not really tied to the donation nor implies we're involved in the specification. 😄 

> Is your spec core team seat "in the name of tb"?

Not really, no. I'm mostly there to represent servers.

Reply:
> Right, which is actually not really tied to the donation nor implies we're involved in the specification.

Do we want to remove this mention entirely then?